src/kernel.rs

```diff
@@ -182,6 +182,38 @@ impl<'res> DescriptorSet<'res> {
 
         self
     }
+
+    /// Binds a [`Sampler`] as a sampler object in the shader.
+    /// ### Example WGSL syntax:

```

Comment:
Note: No GLSL example added because AFAIK GLSL does not have separate sampler and texture objects. They are bundled together into combined objects that occupy the same texture slot.

Reply:
Just wondering - does it cause a problem using SPIRV coming from GLSL shaders?